### ISSUE TITLE
How to find the right shift

### ISSUE THREAD

**Issue by t-a-m-i** [Teichlab/bin2cell#8]
Hi all, 
Great work you´ve done with bin2cell! I´d like to use it for my project and was running your notebook "N1_demo_b2c_crc_submission". Here you adjust a slight shift in coordinates. I have two questions: 

1. You don´t do that in the mouse brain notebook. How would I know if I need to adjust for such a shift in my data? It seems very subtle and I wouldn´t have thought to adjust anything myself. 
2. How do you come up with the exact values to apply to the data? "if you found the right shift, apply it to the full object as well" from the notebook seems a bit vague to me. 

Thank you for your help!

**Comment by nadavyayon** [comment]
hello, so to answer your questions

1) yes in the mouse brain we found the coordinated to be nicely aligned. This is however a major challenge in visiumHD as the alignment is done in 2 steps as described here https://www.10xgenomics.com/support/software/space-ranger/latest/analysis/inputs/visium-hd-loupe-alignment. But saying all that, this is done on a millimetre scale and not on the micron scale we want in theory for visiumHD. 
So to really know if your expression data (GEX) matches the image (HE) you need to estimate if the shape of the GEX grid matches the HE morphology in high resolution. This is also not simple as the RNA profile doesn't always match the location of the nuclei, in some cases the nuclei express less RNA than their surroundings cytoplasmic regions. in the mouse brain since it's so sparse we can tell quite easily what is a cell both from GEX and HE.  

At some point we even tried to develop a custom alignment algorithm for GEX and HE but it was not consistent across sample types so we scrapped it. 

2) Well unfortunately i don't have a good answer to this but to try and see if after shifting the expression and tissue match.  

Bottom line, 
1) start from making sure the cytassist to fiducial and the image to cytassist image are as aligned as possible and i would strongly recommend following the tutorial from 10X above. 
2) Look at the data as we did in the tutorial and see if further correction is needed
3) Test different shifts and see if this ends up with more consistent cell type predictions although this might be very time consuming to implement 

Hope this answers your question 

N

**Comment by t-a-m-i** [comment]
Thanks a lot for the quick response! this was very insightful and I´ll def have a look at the tutorial from 10X
